### PR TITLE
[dotnet] Adjust logging examples to use File Handler

### PR DESCRIPTION
**Pull request by nvborisenko**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by titusfortner — CHANGES_REQUESTED** [review]
When we update the example files, we also need to update the line references in these files (all translations): https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md?plain=1

**Comment by titusfortner** [comment]
Looks like each of the runners have a different default version of Chrome, I've set it to use Stable by default.

File: website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Each of the bindings have a method in the options class to set this.
 {{< gh-codeblock path="examples/python/tests/drivers/test_remote_webdriver.py#L37-L39" >}}
 {{% /tab %}}
 {{< tab header="CSharp" >}}
-{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L58-L62" >}}
+{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L59-L63" >}}
```
Comment (titusfortner) [inline website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md]:
```suggestion
{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L59-L62" >}}
```
Same for others

File: website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md [hunk 2]
```diff
@@ -159,7 +159,7 @@ Selenium looks for the name of the provided file in the list and downloads it to
 {{< gh-codeblock path="examples/python/tests/drivers/test_remote_webdriver.py#L53" >}}
 {{% /tab %}}
 {{< tab header="CSharp" >}}
-{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L77" >}}
+{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L78" >}}
```
Comment (titusfortner) [inline website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md:162]:
```suggestion
{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L79" >}}
```

File: website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ but you can also delete all files during the session.
 {{< gh-codeblock path="examples/python/tests/drivers/test_remote_webdriver.py#L59" >}}
 {{% /tab %}}
 {{< tab header="CSharp" >}}
-{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L82" >}}
+{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L83" >}}
```
Comment (titusfortner) [inline website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md]:
```suggestion
{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L84" >}}
```

File: website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md [hunk 4]
```diff
@@ -220,7 +220,7 @@ to get all the functionality by default:
 {{% /tab %}}
 {{< tab header="CSharp" >}}
 .NET uses a custom command executor for executing commands that are valid for the given browser in the remote driver.
-{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L94-L98" >}}
+{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L95-L99" >}}
```
Comment (titusfortner) [inline website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md]:
```suggestion
{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L96-L100" >}}
```

File: website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md [hunk 5]
```diff
@@ -134,7 +134,7 @@ so the list is an immediate snapshot of what file names are currently in the dir
 {{< gh-codeblock path="examples/python/tests/drivers/test_remote_webdriver.py#L47" >}}
 {{% /tab %}}
 {{< tab header="CSharp" >}}
-{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L70" >}}
+{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L71" >}}
```
Comment (titusfortner) [inline website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md]:
```suggestion
{{< gh-codeblock path="examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L72" >}}
```

**Review by titusfortner — CHANGES_REQUESTED** [review]
Looks like you added a line here - https://github.com/SeleniumHQ/seleniumhq.github.io/blob/702edf5f619cf48ed31c02d621032b1c1fcef095/examples/dotnet/SeleniumDocs/Drivers/RemoteWebDriverTest.cs#L63

which threw everything off again

File: website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md (hunk 1 above)
Comment (nvborisenko) [reply website_and_docs/content/documentation/webdriver/drivers/remote_webdriver.en.md]: I did it the same as for java.

**Review by titusfortner — APPROVED** [review]

**Pull request merged**